From akpm@osdl.org Tue May  9 10:49:33 2006
Message-Id: <200605091749.k49HnRbn016281@shell0.pdx.osdl.net>
From: Brice Goglin <brice@myri.com>
Subject: PCI: Add PCI_CAP_ID_VNDR
To: brice@myri.com, greg@kroah.com, mm-commits@vger.kernel.org
From: akpm@osdl.org
Date: Tue,  9 May 2006 10:52:09 -0700

From: Brice Goglin <brice@myri.com>

Add the vendor-specific extended capability PCI_CAP_ID_VNDR.  It will be
used by the Myri-10G Ethernet driver (will be submitted soon).

Signed-off-by: Brice Goglin <brice@myri.com>
Signed-off-by: Andrew Morton <akpm@osdl.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 include/linux/pci_regs.h |    1 +
 1 file changed, 1 insertion(+)

--- gregkh-2.6.orig/include/linux/pci_regs.h
+++ gregkh-2.6/include/linux/pci_regs.h
@@ -197,6 +197,7 @@
 #define  PCI_CAP_ID_CHSWP	0x06	/* CompactPCI HotSwap */
 #define  PCI_CAP_ID_PCIX	0x07	/* PCI-X */
 #define  PCI_CAP_ID_HT_IRQCONF	0x08	/* HyperTransport IRQ Configuration */
+#define  PCI_CAP_ID_VNDR	0x09	/* Vendor specific capability */
 #define  PCI_CAP_ID_SHPC 	0x0C	/* PCI Standard Hot-Plug Controller */
 #define  PCI_CAP_ID_EXP 	0x10	/* PCI Express */
 #define  PCI_CAP_ID_MSIX	0x11	/* MSI-X */
